lib/tasks/product/update_name.rake

```diff
@@ -0,0 +1,7 @@
+namespace :product do
+  desc 'Assigns first category to all products without Category'
+  task :port_legacy_products => [ :environment ] do 
+    category = Category.first
+    Product.where(category: nil).update!(category: category) if category
```

Comment:
use some output for user as well how much products were updated
and rename file as well